auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -522,6 +532,8 @@ import AffiliationInvitationService from '@/services/affiliation-invitation.serv
 import { AffiliationInvitationStatus } from '@/models/affiliation'
 import { Base64 } from 'js-base64'
 import ConfigHelper from '@/util/config-helper'
+import StartNewBusinessHelp from '@/components/auth/manage-business/StartNewBusinessHelp.vue'
+import ExpandableHelp from '@/components/auth/common/ExpandableHelp.vue'
```

Comment:
Please fix the import order.

Don't worry about the other lint warnings since you didn't touch those files.